### DISCUSSION TITLE
Vaex datafame heatmap colormap question

### DISCUSSION THREAD

**Discussion by fqking** [vaexio/vaex#2407] [General]
I have a csv file and loaded into vaex datafame. The left plot is the heatmap of two columns and the right plot is the same two columns with the rectangular filter.  How can I make the same colormap intensity in the right plot?  
![image](https://github.com/vaexio/vaex/assets/6226024/f613dd71-a1a2-45da-88f8-27e3b914e485)
I wanted something like this in the right plot. 
![image](https://github.com/vaexio/vaex/assets/6226024/d5aa86d9-78b2-4ff8-91e4-d9ef963bf769)

**Comment by fqking** [comment]
Thank you!

**Reply by loveyylele** [reply]
NB
